runtime/str.go

```diff
@@ -157,6 +157,29 @@ func strAdd(f *Frame, v, w *Object) (*Object, *BaseException) {
 	return NewStr(stringV + stringW).ToObject(), nil
 }
 
+func strCapitalize(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	if raised := checkMethodArgs(f, "capitalize", args, StrType); raised != nil {
+		return nil, raised
+	}
+	s := toStrUnsafe(args[0]).Value()
+	buf := bytes.Buffer{}
+	runes := []rune(s)
+	numRunes := len(runes)
+	writeRune := func(r rune, transform func(r rune) rune) {
+		if r <= unicode.MaxASCII {
+			r = transform(r)
+		}
+		buf.WriteRune(r)
+	}
+	if numRunes > 0 {
+		writeRune(runes[0], func(r rune) rune { return unicode.ToUpper(r) })
+	}
+	for i := 1; i < numRunes; i++ {
```

Comment:
I like the simplicity of this and agree something like this would be best. But again, I think the trouble is that the Go standard library will affect UTF-8 characters different than Python does for str. In Python, str is just bytes and it seems like it only affects ASCII characters. The ToLower() part of the above will cause multi-byte characters like the last two in "ВОЛ" to switch to their lowercase equivalents, which wouldn't happen for a UTF-8 encoded equivalent str in CPython.